updates/3/updates.sql

```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("LNCHX", "S", "LaunchX", "LNCHX", 1620720017, NULL, "launchx", "", "LNCHX"); INSERT INTO common_asset_details(asset_id, forked) VALUES("LNCHX", NULL);
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("POLX", "S", "Polylastic", "POLX", 1619730927, NULL, "polylastic", "POLX", "POLX"); INSERT INTO common_asset_details(asset_id, forked) VALUES("POLX", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3", "C", "Fractal Protocol Token", "FCL", 1613480889, NULL, "fractal", "FCL", "0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xbDEC45952B5E234EdDC2981B43eeD360826D5087", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xbDEC45952B5E234EdDC2981B43eeD360826D5087", "C", "Mogu Token", "MOGX", 1566834869, NULL, "mogu", "MOGU", "0xbDEC45952B5E234EdDC2981B43eeD360826D5087");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5273063725a43A323300C502478C22FbB4e92C2D", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x5273063725a43A323300C502478C22FbB4e92C2D", "C", "MY IDENTITY COIN", "MYID", 1614319404, NULL, "my-identity-coin", "MYID", "0x5273063725a43A323300C502478C22FbB4e92C2D");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("NDAU", "W", "Ndau", "NDAU", 1569891600, NULL, "ndau", "XND", "NDAU"); INSERT INTO common_asset_details(asset_id, forked) VALUES("NDAU", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", "C", "Polka City", "POLC", 1613575886, NULL, "polkacity", "", "0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("WIN-4", "P", "WINkLink", "WIN", 1564440570, NULL, "winklink", "", "WIN-4"); INSERT INTO common_asset_details(asset_id, forked) VALUES("WIN-4", NULL);
```

Comment:
I think this is also already in the DB as `WIN-3`:

```
sqlite> SELECT * from assets WHERE symbol="WIN";
_ceth_0xBfaA8cF522136C6FAfC1D53Fe4b85b4603c765b8|C|WinToken|WIN|1528126931|||WINT|0xBfaA8cF522136C6FAfC1D53Fe4b85b4603c765b8
_ceth_0x899338b84D25aC505a332aDCE7402d697D947494|C|WCOIN|WIN|1514073600||wawllet|WCOIN|0x899338b84D25aC505a332aDCE7402d697D947494
WIN-3|P|WINk|WIN|1562532570||wink|WIN|WIN-3
```

Reply:
I saw a different coingecko id and assumed they were different. Seems like win redirects winlink